ner_v2/detectors/temporal/time/standard_time_regex.py

```diff
@@ -12,14 +12,14 @@
 
 
 class BaseRegexTime(object):
-    def __init__(self, entity_name, data_directory_path, timezone='UTC', is_past_referenced=False):
+    def __init__(self, entity_name, data_directory_path, timezone='UTC', past_time_referenced=False):
```

Comment:
This should also accept `range_enabled` and `form_check` arguments as `en` detector accepts them